### PR TITLE
Adds a new prefab: prefab_saferoom

### PR DESCRIPTION
**Pull request by MetricDuck**
[FEATURE] [MAPPING] [SPRITES]
## About the PR 
This prefab is my submission for the 'April' Goonstation Monthly Contest (https://forum.ss13.co/showthread.php?tid=16992).

It's a small, puzzle based prefab with a bit of a story attached (I was going for something along the lines of walking into the aftermath of a BM/TZ episode, YMMV on whether I achieved it!).

Concrete stats:
- Size: 33x22
- Probability: 15
- Hazards: Environmental (mitigatable)
- New loot: Health scan implant that remote clonescans with a chance of adding puritan (See note).
- Other loot & considerations: Some  drugs & cash. Also prefab contains most of a pre-built functioning lab/bar which might have balance implications.

Please note I've never done anything in BYOND before, so I had to learn as I went! Super grateful for any constructive criticism - I read the guidelines, but I have particular concerns about:
- File organisation. Where am I putting stuff? I've erred on the side of caution and contained everything within a small number of files for now.
- Coding. Particularly catching nulls/runtimes.
- Respriting. Do the little fake objects look good enough?
- Probably like, a hundred other things!!!!

(Note) The implant was mostly for story/gameplay integration, I am more than happy to think of something else if it is verboten! 

## Why's this needed?
Contest submission. Also I've always wanted to contribute something to the server and this seemed like a good opportunity.

## Changelog
```changelog
(u)MetricDuck
(*)Added a new space prefab, a seemingly abandoned safe room built into the side of an asteroid.
```

### CODE REVIEW

**Review by Tarmunora — CHANGES_REQUESTED** [review]
You need to push your modified `goonstation.dme`, as you're adding a new file

File: code/z_adventurezones/safehouse.dm [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+		bioHolder.Uid = bioHolder.CreateUid()
+		bioHolder.uid_hash = md5(bioHolder.Uid)
+		. = ..()
+
+	New()
+		..()
+
+		SPAWN_DBG(5 SECONDS)
+		for(var/mob/living/carbon/human/dead_exec/M)
+			qdel(M) //Jean's only here to set up the puzzle, we don't want them hanging around.
```
Comment (ZeWaka) [inline code/z_adventurezones/safehouse.dm:121]: this bit of code makes no sense

File: code/z_adventurezones/safehouse.dm [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+		src.setup_blood_bioHolder()
+		src.setup_bio_handscanner_access()
+		src.setup_bot_master()
+		qdel(src)
+
+	proc/setup_blood_bioHolder()
+		for(var/obj/item/reagent_containers/iv_drip/O in src.loc)
+			if(!O.reagents.has_reagent("blood"))
+				return
+			for(var/mob/living/carbon/human/dead_exec/M in world) //aka Jean
```
Comment (ZeWaka) [inline code/z_adventurezones/safehouse.dm:155]: why not just spawn the dude inside the object with a variable then delete him

**Comment by MetricDuck** [comment]
I made a mistake and tried to remerge from master mid-changes!! Sorry! Still getting the hang of this
Not sure why it's requested a review from UrsulaMajor but I only wanted to change two easy fixes before I went to bed!!

**Comment by MetricDuck** [comment]
Closing to resubmit, I'll transcribe comments

**Pull request closed without merging**